src/components/DatasetCard.vue

```diff
@@ -8,17 +8,23 @@
         </span>
         <div class="card-right" >
           <div class="title" @click="cardClicked">{{entry.description}}</div>
-          <div class="details">{{contributors}}</div>
```

Comment:
Can you reverse the change here please? Modify the computed contributors method instead.

Reply:
Of course! Done in commit ddf0d49.

Reply:
@alan-wu, I believe this has been addressed, so feel free to resolve the conversation (to make things "clean") if you are happy with the "fix".